tiledb/sm/tile/generic_tile_io.cc

```diff
@@ -58,6 +59,30 @@ GenericTileIO::GenericTileIO(ContextResources& resources, const URI& uri)
 /*               API              */
 /* ****************************** */
 
+tuple<Status, optional<Tile>> GenericTileIO::load(
```

Comment:
Let's prefer to throw rather than return Status for new functions (and as much as feasible for refactoring).

Reply:
I'll give it a whirl.

Reply:
Fixed.